test/cut/test_cut_set.py

```diff
@@ -193,6 +193,15 @@ def test_trim_to_supervisions_mixed_cuts():
     assert cut.supervisions[0].id == 'sup4'
 
 
+def is_pandas_installed():
```

Comment:
How about using the approach from torchaudio?
https://github.com/pytorch/audio/blob/6bad3a66a7a1c7cc05755e9ee5931b7391d2b94c/torchaudio/_internal/module_utils.py#L9
```python
def is_module_available(*modules: str) -> bool:
    r"""Returns if a top-level module with :attr:`name` exists *without**
    importing it. This is generally safer than try-catch block around a
    `import X`. It avoids third party libraries breaking assumptions of some of
    our tests, e.g., setting multiprocessing start method when imported
    (see librosa/#747, torchvision/#544).
    """
    return all(importlib.util.find_spec(m) is not None for m in modules)
```

It is more general.

Reply:
Great! Cool find.